lib/fastlane/plugin/wpmreleasetoolkit/actions/common/removebranchprotection_action.rb

```diff
@@ -14,7 +14,9 @@ def self.run(params)
         branch_prot[:enforce_admins] = nil
         branch_prot[:required_pull_request_reviews] = { url: "#{branch_url}/protection/required_pull_request_reviews", dismiss_stale_reviews: false, require_code_owner_reviews: false }
 
-        Fastlane::Helper::GithubHelper.github_client().unprotect_branch(repository, branch_name, branch_prot)
+        access_token = params[:access_token]
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: access_token)
+        github_helper.unprotect_branch(repository, branch_name, branch_prot)
```

Comment:
method of `Octokit::Client` not `GithubHelper`

Reply:
Fixed 👍